models/tasks.js

```diff
@@ -44,9 +44,17 @@ const updateTask = async (taskData, taskId = null) => {
  *
  * @return {Promise<tasks|Array>}
  */
-const fetchTasks = async () => {
+const fetchTasks = async (limit, offset) => {
+  let lastDoc;
+  if (offset) {
+    lastDoc = await tasksModel.doc(offset).get();
+    lastDoc = lastDoc.data();
+  }
+  const limitDocuments = Number(limit);
```

Comment:
Can we request any number of documents from Firestore? Should there be an upper limit here?